sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -185,6 +230,40 @@ export interface ManifestPageResponse extends Array<ArtifactManifestProperties>
     continuationToken?: string;
 }
 
+// @public
+export interface OciAnnotations {
+    [additionalProperties: string]: unknown;
```

Comment:
The only advantage is it would make it a type error to access something that is unknown on the root object, e.g. otherwise `annotations.uri` wouldn't be flagged even though it's `annotations.url`

Reply:
good point!

Reply:
Yeah good point, I do wonder whether it's worth the extra wrapping/unwrapping to do it like that though?

Reply:
I'm fine doing a beta with it like this and collecting feedback. It's burned us in the past, but that was mostly on options types